NOtClassed/infinity_traj_parameters.txt
{"t": [0.0, 0.025, 0.05, 0.07500000000000001, 0.1, 0.125, 0.15000000000000002, 0.17500000000000002, 0.2, 0.225, 0.25, 0.275, 0.30000000000000004, 0.325, 0.35000000000000003, 0.375, 0.4, 0.42500000000000004, 0.45, 0.47500000000000003, 0.5, 0.525, 0.55, 0.5750000000000001, 0.6000000000000001, 0.625, 0.65, 0.675, 0.7000000000000001, 0.7250000000000001, 0.75, 0.775, 0.8, 0.8250000000000001, 0.8500000000000001, 0.875, 0.9, 0.925, 0.9500000000000001, 0.9750000000000001, 1.0, 1.0250000000000001, 1.05, 1.075, 1.1, 1.125, 1.1500000000000001, 1.175, 1.2000000000000002, 1.225, 1.25, 1.2750000000000001, 1.3, 1.3250000000000002, 1.35, 1.375, 1.4000000000000001, 1.425, 1.4500000000000002, 1.475, 1.5, 1.5250000000000001, 1.55, 1.5750000000000002, 1.6, 1.625, 1.6500000000000001, 1.675, 1.7000000000000002, 1.725, 1.75, 1.7750000000000001, 1.8, 1.8250000000000002, 1.85, 1.875, 1.9000000000000001, 1.925, 1.9500000000000002, 1.975, 2.0, 2.025, 2.0500000000000003, 2.075, 2.1, 2.125, 2.15, 2.1750000000000003, 2.2, 2.225, 2.25, 2.275, 2.3000000000000003, 2.325, 2.35, 2.375, 2.4000000000000004, 2.4250000000000003, 2.45, 2.475, 2.5, 2.5250000000000004, 2.5500000000000003, 2.575, 2.6, 2.625, 2.6500000000000004, 2.6750000000000003, 2.7, 2.725, 2.75, 2.7750000000000004, 2.8000000000000003, 2.825, 2.85, 2.875, 2.9000000000000004, 2.9250000000000003, 2.95, 2.975, 3.0, 3.0250000000000004, 3.0500000000000003, 3.075, 3.1, 3.125, 3.1500000000000004, 3.1750000000000003, 3.2, 3.225, 3.25, 3.2750000000000004, 3.3000000000000003, 3.325, 3.35, 3.375, 3.4000000000000004, 3.4250000000000003, 3.45, 3.475, 3.5, 3.5250000000000004, 3.5500000000000003, 3.575, 3.6, 3.625, 3.6500000000000004, 3.6750000000000003, 3.7, 3.725, 3.75, 3.7750000000000004, 3.8000000000000003, 3.825, 3.85, 3.875, 3.9000000000000004, 3.9250000000000003, 3.95, 3.975, 4.0, 4.025, 4.05, 4.075, 4.1000000000000005, 4.125, 4.15, 4.175, 4.2, 4.2250000000000005, 4.25, 4.275, 4.3, 4.325, 4.3500000000000005, 4.375, 4.4, 4.425, 4.45, 4.4750000000000005, 4.5, 4.525, 4.55, 4.575, 4.6000000000000005, 4.625, 4.65, 4.675, 4.7, 4.7250000000000005, 4.75, 4.775, 4.800000000000001, 4.825, 4.8500000000000005, 4.875, 4.9, 4.925000000000001, 4.95, 4.9750000000000005, 5.0, 5.025, 5.050000000000001, 5.075, 5.1000000000000005, 5.125, 5.15, 5.175000000000001, 5.2, 5.2250000000000005, 5.25, 5.275, 5.300000000000001, 5.325, 5.3500000000000005, 5.375, 5.4, 5.425000000000001, 5.45, 5.4750000000000005, 5.5, 5.525, 5.550000000000001, 5.575, 5.6000000000000005, 5.625, 5.65, 5.675000000000001, 5.7, 5.7250000000000005, 5.75, 5.775, 5.800000000000001, 5.825, 5.8500000000000005, 5.875, 5.9, 5.925000000000001, 5.95, 5.9750000000000005, 6.0, 6.025, 6.050000000000001, 6.075, 6.1000000000000005, 6.125, 6.15, 6.175000000000001, 6.2, 6.2250000000000005, 6.25, 6.275, 6.300000000000001, 6.325, 6.3500000000000005, 6.375, 6.4, 6.425000000000001, 6.45, 6.4750000000000005, 6.5, 6.525, 6.550000000000001, 6.575, 6.6000000000000005, 6.625, 6.65, 6.675000000000001, 6.7, 6.7250000000000005, 6.75, 6.775, 6.800000000000001, 6.825, 6.8500000000000005, 6.875, 6.9, 6.925000000000001, 6.95, 6.9750000000000005, 7.0, 7.025, 7.050000000000001, 7.075, 7.1000000000000005, 7.125, 7.15, 7.175000000000001, 7.2, 7.2250000000000005, 7.25, 7.275, 7.300000000000001, 7.325, 7.3500000000000005, 7.375, 7.4, 7.425000000000001, 7.45, 7.4750000000000005, 7.5, 7.525, 7.550000000000001, 7.575, 7.6000000000000005, 7.625, 7.65, 7.675000000000001, 7.7, 7.7250000000000005, 7.75, 7.775, 7.800000000000001, 7.825, 7.8500000000000005, 7.875, 7.9, 7.925000000000001, 7.95, 7.9750000000000005, 8.0, 8.025, 8.05, 8.075000000000001, 8.1, 8.125, 8.15, 8.175, 8.200000000000001, 8.225, 8.25, 8.275, 8.3, 8.325000000000001, 8.35, 8.375, 8.4, 8.425, 8.450000000000001, 8.475, 8.5, 8.525, 8.55, 8.575000000000001, 8.6, 8.625, 8.65, 8.675, 8.700000000000001, 8.725, 8.75, 8.775, 8.8, 8.825000000000001, 8.85, 8.875, 8.9, 8.925, 8.950000000000001, 8.975, 9.0, 9.025, 9.05, 9.075000000000001, 9.1, 9.125, 9.15, 9.175, 9.200000000000001, 9.225, 9.25, 9.275, 9.3, 9.325000000000001, 9.35, 9.375, 9.4, 9.425, 9.450000000000001, 9.475, 9.5, 9.525, 9.55, 9.575000000000001, 9.600000000000001, 9.625, 9.65, 9.675, 9.700000000000001, 9.725000000000001, 9.75, 9.775, 9.8, 9.825000000000001, 9.850000000000001, 9.875, 9.9, 9.925, 9.950000000000001, 9.975000000000001, 10.0, 10.025, 10.05, 10.075000000000001, 10.100000000000001, 10.125, 10.15, 10.175, 10.200000000000001, 10.225000000000001, 10.25, 10.275, 10.3, 10.325000000000001, 10.350000000000001, 10.375, 10.4, 10.425, 10.450000000000001, 10.475000000000001, 10.5, 10.525, 10.55, 10.575000000000001, 10.600000000000001, 10.625, 10.65, 10.675, 10.700000000000001, 10.725000000000001, 10.75, 10.775, 10.8, 10.825000000000001, 10.850000000000001, 10.875, 10.9, 10.925, 10.950000000000001, 10.975000000000001, 11.0, 11.025, 11.05, 11.075000000000001, 11.100000000000001, 11.125, 11.15, 11.175, 11.200000000000001, 11.225000000000001, 11.25, 11.275, 11.3, 11.325000000000001, 11.350000000000001, 11.375, 11.4, 11.425, 11.450000000000001, 11.475000000000001, 11.5, 11.525, 11.55, 11.575000000000001, 11.600000000000001, 11.625, 11.65, 11.675, 11.700000000000001, 11.725000000000001, 11.75, 11.775, 11.8, 11.825000000000001, 11.850000000000001, 11.875, 11.9, 11.925, 11.950000000000001, 11.975000000000001, 12.0, 12.025, 12.05, 12.075000000000001, 12.100000000000001, 12.125, 12.15, 12.175, 12.200000000000001, 12.225000000000001, 12.25, 12.275, 12.3, 12.325000000000001, 12.350000000000001, 12.375, 12.4, 12.425, 12.450000000000001, 12.475000000000001, 12.5, 12.525, 12.55, 12.575000000000001, 12.600000000000001, 12.625, 12.65, 12.675, 12.700000000000001, 12.725000000000001, 12.75, 12.775, 12.8, 12.825000000000001, 12.850000000000001, 12.875, 12.9, 12.925, 12.950000000000001, 12.975000000000001, 13.0, 13.025, 13.05, 13.075000000000001, 13.100000000000001, 13.125, 13.15, 13.175, 13.200000000000001, 13.225000000000001, 13.25, 13.275, 13.3, 13.325000000000001, 13.350000000000001, 13.375, 13.4, 13.425, 13.450000000000001, 13.475000000000001, 13.5, 13.525, 13.55, 13.575000000000001, 13.600000000000001, 13.625, 13.65, 13.675, 13.700000000000001, 13.725000000000001, 13.75, 13.775, 13.8, 13.825000000000001, 13.850000000000001, 13.875, 13.9, 13.925, 13.950000000000001, 13.975000000000001, 14.0, 14.025, 14.05, 14.075000000000001, 14.100000000000001, 14.125, 14.15, 14.175, 14.200000000000001, 14.225000000000001, 14.25, 14.275, 14.3, 14.325000000000001, 14.350000000000001, 14.375, 14.4, 14.425, 14.450000000000001, 14.475000000000001, 14.5, 14.525, 14.55, 14.575000000000001, 14.600000000000001, 14.625, 14.65, 14.675, 14.700000000000001, 14.725000000000001, 14.75, 14.775, 14.8, 14.825000000000001, 14.850000000000001, 14.875, 14.9, 14.925, 14.950000000000001, 14.975000000000001, 15.0, 15.025, 15.05, 15.075000000000001, 15.100000000000001, 15.125, 15.15, 15.175, 15.200000000000001, 15.225000000000001, 15.25, 15.275, 15.3, 15.325000000000001, 15.350000000000001, 15.375, 15.4, 15.425, 15.450000000000001, 15.475000000000001, 15.5, 15.525, 15.55, 15.575000000000001, 15.600000000000001, 15.625, 15.65, 15.675, 15.700000000000001, 15.725000000000001, 15.75, 15.775, 15.8, 15.825000000000001, 15.850000000000001, 15.875, 15.9, 15.925, 15.950000000000001, 15.975000000000001, 16.0, 16.025000000000002, 16.05, 16.075, 16.1, 16.125, 16.150000000000002, 16.175, 16.2, 16.225, 16.25, 16.275000000000002, 16.3, 16.325, 16.35, 16.375, 16.400000000000002, 16.425, 16.45, 16.475, 16.5, 16.525000000000002, 16.55, 16.575, 16.6, 16.625, 16.650000000000002, 16.675, 16.7, 16.725, 16.75, 16.775000000000002, 16.8, 16.825, 16.85, 16.875, 16.900000000000002, 16.925, 16.95, 16.975, 17.0, 17.025000000000002, 17.05, 17.075, 17.1, 17.125, 17.150000000000002, 17.175, 17.2, 17.225, 17.25, 17.275000000000002, 17.3, 17.325, 17.35, 17.375, 17.400000000000002, 17.425, 17.45, 17.475, 17.5, 17.525000000000002, 17.55, 17.575, 17.6, 17.625, 17.650000000000002, 17.675, 17.7, 17.725, 17.75, 17.775000000000002, 17.8, 17.825, 17.85, 17.875, 17.900000000000002, 17.925, 17.95, 17.975, 18.0, 18.025000000000002, 18.05, 18.075, 18.1, 18.125, 18.150000000000002, 18.175, 18.2, 18.225, 18.25, 18.275000000000002, 18.3, 18.325, 18.35, 18.375, 18.400000000000002, 18.425, 18.45, 18.475, 18.5, 18.525000000000002, 18.55, 18.575, 18.6, 18.625, 18.650000000000002, 18.675, 18.7, 18.725, 18.75, 18.775000000000002, 18.8, 18.825, 18.85, 18.875, 18.900000000000002, 18.925, 18.95, 18.975, 19.0, 19.025000000000002, 19.05, 19.075, 19.1, 19.125, 19.150000000000002, 19.175, 19.200000000000003, 19.225, 19.25, 19.275000000000002, 19.3, 19.325000000000003, 19.35, 19.375, 19.400000000000002, 19.425, 19.450000000000003, 19.475, 19.5, 19.525000000000002, 19.55, 19.575000000000003, 19.6, 19.625, 19.650000000000002, 19.675, 19.700000000000003, 19.725, 19.75, 19.775000000000002, 19.8, 19.825000000000003, 19.85, 19.875, 19.900000000000002, 19.925, 19.950000000000003, 19.975, 20.0, 20.025000000000002, 20.05, 20.075000000000003, 20.1, 20.125, 20.150000000000002, 20.175, 20.200000000000003, 20.225, 20.25, 20.275000000000002, 20.3, 20.325000000000003, 20.35, 20.375, 20.400000000000002, 20.425, 20.450000000000003, 20.475, 20.5, 20.525000000000002, 20.55, 20.575000000000003, 20.6, 20.625, 20.650000000000002, 20.675, 20.700000000000003, 20.725, 20.75, 20.775000000000002, 20.8, 20.825000000000003, 20.85, 20.875, 20.900000000000002, 20.925, 20.950000000000003, 20.975, 21.0, 21.025000000000002, 21.05, 21.075000000000003, 21.1, 21.125, 21.150000000000002, 21.175, 21.200000000000003, 21.225, 21.25, 21.275000000000002, 21.3, 21.325000000000003, 21.35, 21.375, 21.400000000000002, 21.425, 21.450000000000003, 21.475, 21.5, 21.525000000000002, 21.55, 21.575000000000003, 21.6, 21.625, 21.650000000000002, 21.675, 21.700000000000003, 21.725, 21.75, 21.775000000000002, 21.8, 21.825000000000003, 21.85, 21.875, 21.900000000000002, 21.925, 21.950000000000003, 21.975, 22.0, 22.025000000000002, 22.05, 22.075000000000003, 22.1, 22.125, 22.150000000000002, 22.175, 22.200000000000003, 22.225, 22.25, 22.275000000000002, 22.3, 22.325000000000003, 22.35, 22.375, 22.400000000000002, 22.425, 22.450000000000003, 22.475, 22.5, 22.525000000000002, 22.55, 22.575000000000003, 22.6, 22.625, 22.650000000000002, 22.675, 22.700000000000003, 22.725, 22.75, 22.775000000000002, 22.8, 22.825000000000003, 22.85, 22.875, 22.900000000000002, 22.925, 22.950000000000003, 22.975, 23.0, 23.025000000000002, 23.05, 23.075000000000003, 23.1, 23.125, 23.150000000000002, 23.175, 23.200000000000003, 23.225, 23.25, 23.275000000000002, 23.3, 23.325000000000003, 23.35, 23.375, 23.400000000000002, 23.425, 23.450000000000003, 23.475, 23.5, 23.525000000000002, 23.55, 23.575000000000003, 23.6, 23.625, 23.650000000000002, 23.675, 23.700000000000003, 23.725, 23.75, 23.775000000000002, 23.8, 23.825000000000003, 23.85, 23.875, 23.900000000000002, 23.925, 23.950000000000003, 23.975, 24.0, 24.025000000000002, 24.05, 24.075000000000003, 24.1, 24.125, 24.150000000000002, 24.175, 24.200000000000003, 24.225, 24.25, 24.275000000000002, 24.3, 24.325000000000003, 24.35, 24.375, 24.400000000000002, 24.425, 24.450000000000003, 24.475, 24.5, 24.525000000000002, 24.55, 24.575000000000003, 24.6, 24.625, 24.650000000000002, 24.675, 24.700000000000003, 24.725, 24.75, 24.775000000000002, 24.8, 24.825000000000003, 24.85, 24.875, 24.900000000000002, 24.925, 24.950000000000003, 24.975, 25.0, 25.025000000000002, 25.05, 25.075000000000003, 25.1, 25.125, 25.150000000000002, 25.175, 25.200000000000003, 25.225, 25.25, 25.275000000000002, 25.3, 25.325000000000003, 25.35, 25.375, 25.400000000000002, 25.425, 25.450000000000003, 25.475, 25.5, 25.525000000000002, 25.55, 25.575000000000003, 25.6, 25.625, 25.650000000000002, 25.675, 25.700000000000003, 25.725, 25.75, 25.775000000000002, 25.8, 25.825000000000003, 25.85, 25.875, 25.900000000000002, 25.925, 25.950000000000003, 25.975, 26.0, 26.025000000000002, 26.05, 26.075000000000003, 26.1, 26.125, 26.150000000000002, 26.175, 26.200000000000003, 26.225, 26.25, 26.275000000000002, 26.3, 26.325000000000003, 26.35, 26.375, 26.400000000000002, 26.425, 26.450000000000003, 26.475, 26.5, 26.525000000000002, 26.55, 26.575000000000003, 26.6, 26.625, 26.650000000000002, 26.675, 26.700000000000003, 26.725, 26.75, 26.775000000000002, 26.8, 26.825000000000003, 26.85, 26.875, 26.900000000000002, 26.925, 26.950000000000003, 26.975, 27.0, 27.025000000000002, 27.05, 27.075000000000003, 27.1, 27.125, 27.150000000000002, 27.175, 27.200000000000003, 27.225, 27.25, 27.275000000000002, 27.3, 27.325000000000003, 27.35, 27.375, 27.400000000000002, 27.425, 27.450000000000003, 27.475, 27.5, 27.525000000000002, 27.55, 27.575000000000003, 27.6, 27.625, 27.650000000000002, 27.675, 27.700000000000003, 27.725, 27.75, 27.775000000000002, 27.8, 27.825000000000003, 27.85, 27.875, 27.900000000000002, 27.925, 27.950000000000003, 27.975, 28.0, 28.025000000000002, 28.05, 28.075000000000003, 28.1, 28.125, 28.150000000000002, 28.175, 28.200000000000003, 28.225, 28.25, 28.275000000000002, 28.3, 28.325000000000003, 28.35, 28.375, 28.400000000000002, 28.425, 28.450000000000003, 28.475, 28.5, 28.525000000000002, 28.55, 28.575000000000003, 28.6, 28.625, 28.650000000000002, 28.675, 28.700000000000003, 28.725, 28.75, 28.775000000000002, 28.8, 28.825000000000003, 28.85, 28.875, 28.900000000000002, 28.925, 28.950000000000003, 28.975, 29.0, 29.025000000000002, 29.05, 29.075000000000003, 29.1, 29.125, 29.150000000000002, 29.175, 29.200000000000003, 29.225, 29.25, 29.275000000000002, 29.3, 29.325000000000003, 29.35, 29.375, 29.400000000000002, 29.425, 29.450000000000003, 29.475, 29.5, 29.525000000000002, 29.55, 29.575000000000003, 29.6, 29.625, 29.650000000000002, 29.675, 29.700000000000003, 29.725, 29.75, 29.775000000000002, 29.8, 29.825000000000003, 29.85, 29.875, 29.900000000000002, 29.925, 29.950000000000003, 29.975, 30.0, 30.025000000000002, 30.05, 30.075000000000003, 30.1, 30.125, 30.150000000000002, 30.175, 30.200000000000003, 30.225, 30.25, 30.275000000000002, 30.3, 30.325000000000003, 30.35, 30.375, 30.400000000000002, 30.425, 30.450000000000003, 30.475, 30.5, 30.525000000000002, 30.55, 30.575000000000003, 30.6, 30.625, 30.650000000000002, 30.675, 30.700000000000003, 30.725, 30.75, 30.775000000000002, 30.8, 30.825000000000003, 30.85, 30.875, 30.900000000000002, 30.925, 30.950000000000003, 30.975, 31.0, 31.025000000000002, 31.05, 31.075000000000003, 31.1, 31.125, 31.150000000000002, 31.175, 31.200000000000003, 31.225, 31.25, 31.275000000000002, 31.3, 31.325000000000003, 31.35, 31.375, 31.400000000000002, 31.425, 31.450000000000003, 31.475, 31.5, 31.525000000000002, 31.55, 31.575000000000003, 31.6, 31.625, 31.650000000000002, 31.675, 31.700000000000003, 31.725, 31.75, 31.775000000000002, 31.8, 31.825000000000003, 31.85, 31.875, 31.900000000000002, 31.925, 31.950000000000003, 31.975, 32.0, 32.025, 32.050000000000004, 32.075, 32.1, 32.125, 32.15, 32.175000000000004, 32.2, 32.225, 32.25, 32.275, 32.300000000000004, 32.325, 32.35, 32.375, 32.4, 32.425000000000004, 32.45, 32.475, 32.5, 32.525, 32.550000000000004, 32.575, 32.6, 32.625, 32.65, 32.675000000000004, 32.7, 32.725, 32.75, 32.775, 32.800000000000004, 32.825, 32.85, 32.875, 32.9, 32.925000000000004, 32.95, 32.975, 33.0, 33.025, 33.050000000000004, 33.075, 33.1, 33.125, 33.15, 33.175000000000004, 33.2, 33.225, 33.25, 33.275, 33.300000000000004, 33.325, 33.35, 33.375, 33.4, 33.425000000000004, 33.45, 33.475, 33.5, 33.525, 33.550000000000004, 33.575, 33.6, 33.625, 33.65, 33.675000000000004, 33.7, 33.725, 33.75, 33.775, 33.800000000000004, 33.825, 33.85, 33.875, 33.9, 33.925000000000004, 33.95, 33.975, 34.0, 34.025, 34.050000000000004, 34.075, 34.1, 34.125, 34.15, 34.175000000000004, 34.2, 34.225, 34.25, 34.275, 34.300000000000004, 34.325, 34.35, 34.375, 34.4, 34.425000000000004, 34.45, 34.475, 34.5, 34.525, 34.550000000000004, 34.575, 34.6, 34.625, 34.65, 34.675000000000004, 34.7, 34.725, 34.75, 34.775, 34.800000000000004, 34.825, 34.85, 34.875, 34.9, 34.925000000000004, 34.95, 34.975, 35.0, 35.025, 35.050000000000004, 35.075, 35.1, 35.125, 35.15, 35.175000000000004, 35.2, 35.225, 35.25, 35.275, 35.300000000000004, 35.325, 35.35, 35.375, 35.4, 35.425000000000004, 35.45, 35.475, 35.5, 35.525, 35.550000000000004, 35.575, 35.6, 35.625, 35.65, 35.675000000000004, 35.7, 35.725, 35.75, 35.775, 35.800000000000004, 35.825, 35.85, 35.875, 35.9, 35.925000000000004, 35.95, 35.975, 36.0], "x": [-6.123233995736766e-15, -2.099875209373433e-10, 2.793220948836866e-06, 2.399040881754413e-05, 9.169862919816406e-05, 0.00024332803317467922, 0.0005226259426017789, 0.0009769374390682143, 0.0016546887621472427, 0.0026030888357569857, 0.003866042807029483, 0.005482269831190156, 0.007483615705231079, 0.00989354971545564, 0.012725834633649407, 0.015983359536465055, 0.019657127264238208, 0.02372539192023186, 0.028152946654367197, 0.03289056765410627, 0.03787462613293955, 0.04302688533698791, 0.04825450325084153, 0.05345026284867761, 0.05849304962597019, 0.0632485902846614, 0.06757045680718682, 0.07130132727662361, 0.07427447979705842, 0.07631548035059166, 0.07724401129823036, 0.07687577639249783, 0.07502441218822989, 0.07150333555614043, 0.06612746275158568, 0.05871474646106867, 0.04908749203507898, 0.037073430880685015, 0.022506545792726734, 0.0052276581232724995, -0.014915201153600038, -0.03806660594544947, -0.06436369992793206, -0.09393627549146305, -0.1269069321619582, -0.16339128372047906, -0.20349818831448813, -0.24732998144683288, -0.29498269726359655, -0.34654626862416826, -0.40210470077501853, -0.461736216949042, -0.5255133768662699, -0.593503170981801, -0.665767094516704, -0.7423612059398197, -0.8233361747660004, -0.9087373234139589, -0.9986046675233753, -1.0929729586481807, -1.1918717326854573, -1.2953253668157563, -1.4033531471561829, -1.5159693487863277, -1.6331833293142477, -1.7549996367134821, -1.881418131786219, -2.0124341252915303, -2.1480385295180704, -2.2882180238727585, -2.4329552338950187, -2.5822289229823574, -2.7360141960199527, -2.8942827140377343, -3.0570029189635664, -3.224140267494429, -3.395657473061726, -3.571514754816005, -3.751670092493324, -3.9360794859512325, -4.124697218070058, -4.317476119606836, -4.514367834468015, -4.7153230837347895, -4.920291926638018, -5.129224016546704, -5.342068849911838, -5.558776006007802, -5.779295375242223, -6.003577373775712, -6.231573142208162, -6.4632347261578476, -6.698515236682637, -6.937368988674505, -7.179751615592541, -7.425620159181724, -7.674933133152273, -7.927650560145694, -8.183733981694788, -8.443146441264322, -8.705852440839514, -8.971817871893172, -9.241009921888685, -9.51339695778109, -9.788948388215852, -10.067634506322952, -10.349426315141013, -10.634295337780356, -10.922213414447977, -11.213152488419231, -11.507084382937194, -11.803980570874232, -12.103811938807404, -12.406548546918154, -12.712159385897671, -13.0206121317587, -13.331872899204082, -13.645905993963527, -13.962673664278393, -14.282135851579635, -14.604249940311826, -14.928970506874974, -15.256249067786174, -15.586033827488189, -15.918269426627385, -16.25289669235909, -16.589852392991098, -16.92906900040264, -17.270474464825046, -17.613992007988642, -17.9595399419824, -18.307031522621273, -18.656374847350342, -19.007472808554223, -19.360223113779174, -19.714518384003117, -20.070246340388966, -20.42729008799161, -20.785528502288678, -21.14483672087412, -21.505086738434986, -21.866148098399705, -22.227888669980917, -22.590175494725965, -22.95287568287162, -23.315857336969973, -23.67899047861815, -24.042147954059125, -24.40520629551375, -24.76804651778152, -25.13055483291435, -25.492623270023266, -25.854150191536174, -26.21504070132645, -26.575206943815875, -26.934568296209452, -27.293051457761194, -27.65059044133741, -28.00712647262253, -28.36260780154352, -28.716989429518996, -29.070232754273487, -29.42230513193381, -29.773179354557744, -30.12283303876232, -30.471247920456037, -30.8184090490411, -31.164303874665528, -31.508921222467567, -31.852250149498524, -32.19427868255608, -32.534992439288914, -32.87437314039407, -33.21239702673478, -33.54903320380343, -33.88424194378756, -34.217972985337205, -34.550163877733816, -34.88073842339566, -35.20960527328054, -35.53665672342408, -35.86176774580782, -36.18480121734299, -36.50563093761683, -36.82414615651152, -37.14025002985354, -37.45385812604063, -37.76489701953888, -38.07330299246815, -38.3790208539741, -38.68200287857421, -38.98220785881767, -39.27960026396623, -39.57414949450437, -39.86582922166217, -40.15461680139186, -40.44049275304828, -40.72344029414042, -41.00344492375983, -41.28049404852876, -41.55457664606294, -41.82568296196872, -42.09380423726365, -42.35893246382467, -42.6210601660294, -42.880180207179585, -43.136285619597366, -43.38936945748132, -43.63942467172065, -43.88644400590796, -44.13041991278128, -44.37134449027887, -44.6092094363174, -44.84400602131834, -45.07572507741662, -45.304357003197104, -45.529891782726025, -45.75231901757729, -45.97162797050375, -46.18780761937108, -46.400846719957464, -46.610733876227236, -46.8174576167074, -47.02100647563561, -47.22136907759881, -47.41853422444679, -47.61249098333996, -47.80322877487119, -47.99073746028997, -48.175007426947595, -48.35602967117308, -48.53379587788218, -48.70829849631002, -48.87953081134514, -49.04748701002409, -49.21216224282358, -49.37355267945877, -49.531655558962804, -49.686469233883045, -49.83799320848489, -49.986228170903125, -50.13117601922556, -50.27283988153301, -50.411224129955045, -50.54633438883196, -50.67817753710169, -50.80676170505441, -50.9320962656205, -51.05419182037649, -51.173060180472845, -51.28871434270344, -51.401168460952, -51.510437813266286, -51.616538764823986, -51.71948872706856, -51.81930611330634, -51.916010291069576, -52.00962153156243, -52.10016095652094, -52.1876504828291, -52.27211276524633, -52.35357113761278, -52.43204955290987, -52.50757252256306, -52.5801650553827, -52.64985259654529, -52.7166609670225, -52.78061630386785, -52.841745001770406, -52.900073656282665, -52.95562900912275, -53.008437895940325, -53.05852719692499, -53.10592379061468, -53.15065451124374, -53.192746109943904, -53.23222522008294, -53.26911832699385, -53.303451742313335, -53.33525158310828, -53.36454375593027, -53.391353945897194, -53.41570761085406, -53.43762998062383, -53.457146061313736, -53.47428064459725, -53.489058321850266, -53.50150350297611, -53.51164043971254, -53.519493253180066, -53.52508596538719, -53.52844253438378, -53.52958689271137, -53.52854298878331, -53.525334830793135, -53.519986532728986, -53.512522362056124, -53.50296678860311, -53.4913445341793, -53.477680622432175, -53.462000428444036, -53.44432972754853, -53.42469474285543, -53.403122190931526, -53.37963932511935, -53.35427397592697, -53.32705458794842, -53.298010252745776, -53.26717073715478, -53.23456650643565, -53.200228741734215, -53.164189351315635, -53.126480975042455, -53.087136981610655, -53.046191458081815, -53.00367919130206, -52.95963564083927, -52.914096903162836, -52.867099666858195, -52.81868115876283, -52.76887908101887, -52.71773153917792, -52.6652769616045, -52.611554010570686, -52.55660148560654, -52.50045821977895, -52.443162969805066, -52.384754300967664, -52.32527046805295, -52.26474929354418, -52.20322804461314, -52.14074331035917, -52.07733088105532, -52.01302563108225, -51.947861407379094, -51.88187092531453, -51.81508567382652, -51.74753583180099, -51.67925019758601, -51.61025613357977, -51.540579527820064, -51.470244774428835, -51.399274774870875, -51.32769096173327, -51.25551334696443, -51.18276059628598, -51.10945013146885, -51.0355982622589, -50.96122034936422, -50.88633100028646, -50.810944299122575, -50.735074071798586, -50.658734187745964, -50.58193889882929, -50.50470321616632, -50.427043324943924, -50.34897703714553, -50.27052428134049, -50.191707628425775, -50.11255285151527, -50.03308951753687, -49.95335160808144, -49.873376683151605, -49.79320044850139, -49.71285550581118, -49.632371551001384, -49.55177557272826, -49.471092049078194, -49.39034314071128, -49.30954887895755, -49.22872734762508, -49.147894857523525, -49.06706611293347, -48.986254369453384, -48.905471582834856, -48.824728548567066, -48.74403503209992, -48.66339988970077, -48.58283118002562, -48.5023362665558, -48.42192191110627, -48.34159435865617, -48.2613594137866, -48.181222509038136, -48.10118876552085, -48.02126304612703, -47.94145000170782, -47.86175411058517, -47.78217971177694, -47.70273103231695, -47.6234122090548, -47.544227305320796, -47.46518032284044, -47.38627520928071, -47.30751586180606, -47.22890612701728, -47.15044979763944, -47.07215060631701, -46.994012216864945, -46.91603821331448, -46.83823208707975, -46.76059722256002, -46.683136881477324, -46.60585418623583, -46.5287521025734, -46.45183342176102, -46.37510074258805, -46.2985564533564, -46.22220271408837, -46.146041439137406, -46.070074280373056, -45.99430261109524, -45.918727510816204, -45.8433497510327, -45.76816978209404, -45.69318772125789, -45.618403342008904, -45.54381606470269, -45.469424948582564, -45.3952286852043, -45.32122559329072, -45.247413615027256, -45.17379031379737, -45.10035287334681, -45.02709809835597, -44.954022416389655, -44.88112188118622, -44.80839217723883, -44.735828625615575, -44.6634261909576, -44.591179489588626, -44.519082798664364, -44.447130066284295, -44.375314922485394, -44.30363069103239, -44.23207040191692, -44.160626804475136, -44.08929238103085, -44.01805936097069, -43.94691973515605, -43.87586527057696, -43.80488752515253, -43.73397786258421, -43.6631274671686, -43.592327358478954, -43.52156840582689, -43.45084134241806, -43.38013677912006, -43.30944521776366, -43.23875706390342, -43.1680626389684, -43.09735219173865, -43.02661590908941, -42.955843925949964, -42.88502633443127, -42.81415319208241, -42.74321452924307, -42.672200355466295, -42.60110066499242, -42.529905441263175, -42.45860466047156, -42.387188294151116, -42.315646310815104, -42.24396867666347, -42.17214535538321, -42.10016630707403, -42.02802148633867, -41.95570083958337, -41.88319430158096, -41.81049179135437, -41.7375832074447, -41.6644584226325, -41.591107278187195, -41.51751957772204, -41.44368508073816, -41.369593495942915, -41.295234474431545, -41.22059760282454, -41.145672396451744, -41.070448292680574, -40.99491464448037, -40.91906071432085, -40.842875668497086, -40.76634857197643, -40.68946838385586, -40.61222395352192, -40.53460401759671, -40.45659719775135, -40.37819199946493, -40.29937681179915, -40.22013990825488, -40.1404694487681, -40.06035348289602, -39.97977995423616, -39.89873670611117, -39.81721148854232, -39.73519196652577, -39.65266572961161, -39.569620302776464, -39.486043158566154, -39.40192173047334, -39.317243427499584, -39.2319956498381, -39.14616580559983, -39.05974132848782, -38.972709696314816, -38.88505845023416, -38.7967752145501, -38.70784771694876, -38.61826380898371, -38.52801148662009, -38.437078910649944, -38.34545442675081, -38.2531265849647, -38.16008415835453, -38.06631616057159, -37.97181186208552, -37.8765608047719, -37.78055281459873, -37.683778012082435, -37.58622682023867, -37.487889969695075, -37.38875850066066, -37.28882376144282, -37.188077403170176, -37.08651137043465, -36.98411788751921, -36.88088943990848, -36.77681875079544, -36.671898752272064, -36.566122550961545, -36.45948338780957, -36.3519745918076, -36.24358952744622, -36.134321535687484, -36.02416386834066, -35.91310961569062, -35.80115162732701, -35.68828242614811, -35.57449411555074, -35.45977827991128, -35.34412587847162, -35.22752738711681, -35.109973726359435, -34.991456471825344, -34.871967817119256, -34.75150053730894, -34.63004795322051, -34.50760389671466, -34.38416267708944, -34.259719048733245, -34.13426818013138, -34.007805624308645, -33.88032729077396, -33.751829419014406, -33.62230855357242,

Classed/params.py
import jax.numpy as jnp
import numpy as np

# Constants
g = 9.81
g_I = jnp.array([0, 0, -g])  # Gravity vector in the inertial frame



# Inertia matrix for the rocket body
m = 100.0
J1 = 1 / 12 * m * (4 + 3 * 0.25 ** 2)
J2 = J1
J3 = 0.5 * m * 0.25 ** 2
J_B = jnp.diag(jnp.array([J1, J2, J3]))
r_T_B = jnp.array([0, 0, -0.5])

# Transform matrices
vector = np.concatenate(([1], -np.ones(3)))
h = 0.025 
# Create the diagonal matrix
T = np.diag(vector)

zeros_row = np.zeros((1, 3))
I = np.eye(3)

# Vertically stack the zeros_row and the identity matrix
H = np.vstack((zeros_row, I))
cos_delta_max = np.cos(np.deg2rad(15))
max_gimbal = 15
tan_delta_max = np.tan(np.deg2rad(max_gimbal))
# MPC parameters
N_mpc = 50
Q = jnp.identity(13)
R = jnp.identity(3)
Max_Thrust = 2*m*g  # Maximum thrust value
Min_Thrust = 0.25*Max_Thrust

tr_radius = 5

rho_0 = 0.0
rho_1 = 0.25
rho_2 = 0.9
alpha = 2.0
beta = 3.2
